### PR TITLE
Shooter Subsystem Updates (re-pulled due to master merge)

### PR DESCRIPTION
**Pull request by red4stone**

### CODE REVIEW

File: src/main/java/frc/robot/commands/utility/SleepCommand.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/*----------------------------------------------------------------------------*/
```
Comment (matt-orlando) [inline src/main/java/frc/robot/commands/utility/SleepCommand.java:1]: a wpilib named "WaitCommand" already exist and does just this :)

File: src/main/java/frc/robot/commands/groups/ShooterRevCommandgroup.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Robot;
+import frc.robot.commands.FeederSafetyReverseCommand;
+
+public class ShooterRevCommandgroup extends SequentialCommandGroup{
+
+    public ShooterRevCommandgroup() {
+        addCommands(
+            Robot.FeederSafetyReverse,
```
Comment (matt-orlando) [inline src/main/java/frc/robot/commands/groups/ShooterRevCommandgroup.java:12]: The FeederSafetyReverse command can be replaced with ParallelDeadlineGroup (with a WaitCommand being the deadline) and a new StartEndCommand.

**Review by matt-orlando — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/groups/ShooterRevCommandgroup.java (hunk 2 above)
Comment (red4stone) [reply src/main/java/frc/robot/commands/groups/ShooterRevCommandgroup.java:12]: ShooterRevCommandGroup is unused anyway, as it causes other issues- deleted file entirely

**Review by red4stone — COMMENTED** [review]
Reviewed all merges

**Review by matt-orlando — APPROVED** [review]

**Pull request merged**